### ISSUE TITLE
Add support for '3bf81300008131fe15597562696b657934d4' token

### ISSUE THREAD

**Issue by Arsca** [PeculiarVentures/fortify#281]
Reader name: Yubico YubiKey OTP+FIDO+CCID
ATR: 3BF81300008131FE15597562696B657934D4

```js
{
   "cards": [{
   	"atr": "3BF81300008131FE15597562696B657934D4",
   	"name": "Token name",
   	"driver": "8A24F617AA68D09B222D6D72FA4A2A4E726B4C6D"
   }],
   "drivers": [{
   	"id": "8A24F617AA68D09B222D6D72FA4A2A4E726B4C6D",
   	"name": "Driver name",
   	"file": {
   		"windows": "path/to/pkcs11.dll",
   		"osx": "path/to/pkcs11.dylib"
   	}
   }]
}
```

[Smart card ATR parsing 3BF81300008131FE15597562696B657934D4](https://smartcard-atr.apdu.fr/parse?ATR=3BF81300008131FE15597562696B657934D4)

**Comment by rmhrisk** [comment]
This appears to be a Yubico Yubikey 4 OTP+CCID.

@microshine please add this to the ATR map to the Yubikey configuration.

**Comment by microshine** [comment]
It's already in `cards.json`.

There was a problem with `cards.json` file parsing in v1.0.27 which was fixed in v1.0.28.
